src/applet.c

```diff
@@ -191,6 +194,7 @@ void brisk_menu_applet_init_settings(BriskMenuApplet *self)
 
         /* Pump applet settings */
         brisk_menu_applet_settings_changed(self->settings, "hot-key", self);
+        brisk_menu_applet_settings_changed(self->settings, "label-text", self);
```

Comment:
You're pumping the settings here despite changing them earlier manually, only do one (the pump)